File: ParadigmasdaProgramacao/atividadeCsharp/atividadeCsharp/Funcionario.cs
using System;

namespace atividadeCsharp
{
    public class Funcionario
    {
        public Funcionario()
        {
        }

        private string nome;
        private double salario;
        private double salarioMedio;

        public double calcularMedia(double primeiroSalario, double segundoSalario)
        {

            salarioMedio = (primeiroSalario + segundoSalario) / 2;

            return salarioMedio;
        }

        public string GetNome()
        {
            return this.nome;
        }

        public void SetNome(string nome)
        {
            this.nome = nome;
        }

        public double GetSalario()
        {
            return salario;
        }

        public void SetSalario(double salario)
        {
            this.salario = salario;
        }

        public double GetSalarioMedio()
        {
            return salarioMedio;
        }

        public void SetSalarioMedio(double salarioMedio)
        {
            this.salarioMedio = salarioMedio;
        }

    }
}

File: ParadigmasdaProgramacao/atividadeJava/src/Questoes/Funcionario.java
package Questoes;

public class Funcionario {

    private String nome;
    private Double salario;
    private Double salarioMedio;

    public Double calcularMedia(Double primeiroSalario, Double segundoSalario) {

        salarioMedio = (primeiroSalario + segundoSalario) / 2;

        return salarioMedio;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public Double getSalario() {
        return salario;
    }

    public void setSalario(Double salario) {
        this.salario = salario;
    }

    public void setSalarioMedio(Double salarioMedio) {
        this.salarioMedio = salarioMedio;
    }

}
